agreements/financial.md

```diff
@@ -32,11 +32,11 @@ All recurring membership fees go to core funds. The board is responsible for set
 
 #### Sponsorship
 
-A recurring sponsorship with . All sponsorships have a written memorandum which outlines the terms of the sponsorship and where funds are allocated.
+A recurring sponsorship with a written memorandum (between the Enspiral Foundation and the sponsor) which outlines the terms of the sponsorship.
 
-Sponsorship monies can be allocated to any fund, as defined by the sponsorship memorandum.
+Sponsorship money is allocated to core funds and controlled by the board, though they are free to delegate control to financially enabled working groups.
```

Comment:
Ah, I see - so if we put something in along the lines of 'board controls core funds but have to follow the terms in any sponsorship agreements' (which was my intention) then that meets your concern @agentlewis ?

Reply:
@joshuavial please see my attempt to write up a clarified sponsorship clause.

Note I still won't agree to this passing atm. 

Reading the whole thing again I basically see everything is going to core and it is very murky as to what is an operational cost. ie there are going to be more funds put into core now, so how will spend priority be achieved?

Reply:
@agentlewis does the new language addresses this issue?

```Sponsorship money is allocated to the Core Fund and the board is responsible for managing, dispersing, delegating or spending the funds in accordance with the Sponsorship Memorandum. ```

The way i'm reading it nothing has functionally changed for sponsors - they still get to determine where their funds go, it is just that the board is the counterparty to that agreement instead of people from a financially enabled working group.

So if a sponsor wants to defer decision making on where the funds go then the board sets the priorities and if the sponsor wants to allocate funds in a detailed way then they just put that in their sponsorship agreement.

Reply:
Yep that makes sense. 

The missing piece now is for the there to be clarity on how the board will determine spend priority and report on that for funds deferred to them (see my loomio comment which talks about this). That could be out of scope of this agreement, or the agreement can say the board is responsible for engaging with community to determine strategic spend priority.

Reply:
I added something to that effect later in the document.